src/ServicePulse.Host/app/css/particular.css

```diff
@@ -390,7 +391,7 @@ h3 {
     border-radius: 4px;
     color: #777f7f;
     display: block;
-    height: 130px;
+//    height: 130px;
```

Comment:
@WojcikMike I'll fix this once you're done with your changes. Or you can just remove the commented out line yourself.